package.json

```diff
@@ -91,12 +91,13 @@
 		"@actions/core": "1.9.1",
 		"@actions/github": "5.0.3",
 		"@automattic/color-studio": "2.5.0",
-		"@babel/cli": "7.18.9",
-		"@babel/core": "7.18.9",
-		"@babel/plugin-proposal-class-properties": "7.18.6",
-		"@babel/plugin-syntax-jsx": "7.18.6",
-		"@babel/polyfill": "7.12.1",
-		"@babel/preset-typescript": "7.18.6",
+		"@automattic/webpack-rtl-plugin": "^6.0.0",
+		"@babel/cli": "^7.20.7",
+		"@babel/core": "^7.20.7",
+		"@babel/plugin-proposal-class-properties": "^7.18.6",
+		"@babel/plugin-syntax-jsx": "^7.18.6",
+		"@babel/polyfill": "^7.12.1",
+		"@babel/preset-typescript": "^7.18.6",
```

Comment:
If I remember correctly, we use fixed versions in `package.json` (_aka_, without the `^` caret operator), because renovatebot already takes care of updating the dependencies.

Reply:
> Really exciting to see the bundle size improvements webpack 5 brings! 🥳
Yeahh, that's really awesome to see the bundle size decreasing that much 🎉

> I did a partial review so far, and it looks promising. However, I noticed Storybook is broken (`npm run storybook`), even though in `trunk` it worked fine. Can you reproduce as well?
Thank you for pointing this out! I just fixed that. By default, Storybook uses Webpack 4 to build everything, [so I tweaked its configurations](https://storybook.js.org/docs/react/builders/webpack#webpack-5) to make it use Webpack 5


> If I remember correctly, we use fixed versions in `package.json` (_aka_, without the `^` caret operator), because renovatebot already takes care of updating the dependencies.
Great, I just removed the caret operator from the versions there. Thank you!